projects/angular/src/data/tree-view/tree.ts

```diff
@@ -35,30 +35,30 @@ import { ClrTreeNode } from './tree-node';
   host: {
     tabindex: '0',
     '[attr.role]': '"tree"',
-    '[attr.aria-multiselectable]': 'isMultiSelectable',
   },
 })
 export class ClrTree<T> implements AfterContentInit, OnDestroy {
   @ContentChildren(ClrTreeNode) private rootNodes: QueryList<ClrTreeNode<T>>;
 
   private subscriptions: Subscription[] = [];
+  private _isMultiSelectable = false;
 
   constructor(
     public featuresService: TreeFeaturesService<T>,
     private focusManagerService: TreeFocusManagerService<T>,
-    { nativeElement }: ElementRef<HTMLElement>,
-    renderer: Renderer2,
+    private renderer: Renderer2,
+    private el: ElementRef<HTMLElement>,
     ngZone: NgZone
   ) {
     const subscription = ngZone.runOutsideAngular(() =>
-      fromEvent(nativeElement, 'focusin').subscribe((event: FocusEvent) => {
-        if (event.target === nativeElement) {
+      fromEvent(this.el.nativeElement, 'focusin').subscribe((event: FocusEvent) => {
```

Comment:
```suggestion
      fromEvent(el.nativeElement, 'focusin').subscribe((event: FocusEvent) => {
```

The `this` keyword is not needed in the constructor.

Reply:
Please update all usages in the constructor.

Reply:
We can, but in this specific constructor we use `this` for the `focusManagerService`.
Also we use `this` in every constructor for `private` or `public` fields.

Reply:
Okay, that's fine.

Reply:
I opened a separate PR to clean this up everywhere. #1566